<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="aika_teksti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aika</string>
    </property>
   </widget>
   <widget class="QLabel" name="pituus_teksti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pituus</string>
    </property>
   </widget>
   <widget class="QLabel" name="paino_teksti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Paino</string>
    </property>
   </widget>
   <widget class="QLabel" name="lampo_teksti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lämpötila</string>
    </property>
   </widget>
   <widget class="QLabel" name="sahko_teksti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sähkövirta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aika_nappi">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Aloita ajanlaskenta</string>
    </property>
   </widget>
   <widget class="QComboBox" name="pituus_valinta_si">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>mm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>m</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>km</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="aika_valinta_si">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>h</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>d</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>a</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="paino_valinta_si">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>g</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>kg</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="aika_valinta_sk">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>M</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>d</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>a</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="lampo_valinta_si">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>K</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>°C</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="sahko_valinta_si">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="aika_kentta_sk">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aika_kentta_si">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pituus_kentta_si">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pituus_kentta_sk">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="paino_kentta_si">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="paino_kentta_sk">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lampo_kentta_si">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lampo_kentta_sk">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sahko_kentta_si">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sahko_kentta_sk">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aika_kentta_hex">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pituus_kentta_hex">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="paino_kentta_hex">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lampo_kentta_hex">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sahko_kentta_hex">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sahko_valinta_sk">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>R</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tR</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="lampo_valinta_sk">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>G</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tG</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="paino_valinta_sk">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>T</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tT</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="pituus_valinta_sk">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>S</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tS</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="ohje_teksti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Ohjeet: Kirjoita arvo mihin tahansa laatikkoon. Ohjelma muuntaa sen toiseksi yksiköksi ja päivittää arvot. ”Aloita ajanlaskenta” nollaa aika-kentät ja aloittaa ajan laskemisen. ”Viimeinen päivä” asettaa kyseisen rivin päivämäärän annetun vuoden viimeiseksi päiväksi ja ajaksi.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pvm__teksti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Päivämäärä</string>
    </property>
   </widget>
   <widget class="QLabel" name="pvm_sy_teksti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pvm (sy)</string>
    </property>
   </widget>
   <widget class="QLabel" name="pvm_sk_teksti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pvm (sk)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm__kentta_kk">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm__kentta_pp">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm__kentta_vvvv">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_syt_kentta_vvvv">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_syt_kentta_ppp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sy_kentta_v">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sy_kentta_ss">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sk_kentta_rm">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sy_kentta_kk">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sy_kentta_mm">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sy_kentta_pp">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm__kentta_ss">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm__kentta_mm">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sy_kentta_hh">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm__kentta_hh">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sk_kentta_nm">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sk_kentta_mm">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>450</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sk_kentta_kk">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sk_kentta_v">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sk_kentta_tm">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>450</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sk_kentta_pp">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pvm_syt_teksti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pvm (syt)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nopeus_kentta_si">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nopeus_kentta_sk">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="nopeus_valinta_si">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>m/s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>km/h</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="nopeus_valinta_sk">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>S/M</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="nopeus_kentta_hex">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nopeus_teksti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nopeus</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_sk_kentta_m">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>450</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pvm__nappi_viimeinen">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Viimeinen päivä</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pvm_sy_nappi_viimeinen">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Viimeinen päivä</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pvm_sk_nappi_viimeinen">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Viimeinen päivä</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_skhex_kentta_kk">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_skhex_kentta_rm">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>470</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_skhex_kentta_pp">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>470</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_skhex_kentta_mm">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_skhex_kentta_tm">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_skhex_kentta_nm">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>470</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_skhex_kentta_v">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pvm_skhex_kentta_m">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>470</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pvm_skhex_teksti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pvm (sk, hex)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="valikko">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="valikko_tiedosto">
    <property name="title">
     <string>Tiedosto</string>
    </property>
    <addaction name="toiminto_nollaa"/>
    <addaction name="toiminto_lopeta"/>
   </widget>
   <addaction name="valikko_tiedosto"/>
  </widget>
  <widget class="QStatusBar" name="tilapalkki"/>
  <action name="toiminto_nollaa">
   <property name="text">
    <string>Nollaa</string>
   </property>
  </action>
  <action name="toiminto_lopeta">
   <property name="text">
    <string>Lopeta</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pituus_kentta_si</tabstop>
  <tabstop>pituus_valinta_si</tabstop>
  <tabstop>pituus_kentta_sk</tabstop>
  <tabstop>pituus_valinta_sk</tabstop>
  <tabstop>pituus_kentta_hex</tabstop>
  <tabstop>nopeus_kentta_si</tabstop>
  <tabstop>nopeus_valinta_si</tabstop>
  <tabstop>nopeus_kentta_sk</tabstop>
  <tabstop>nopeus_valinta_sk</tabstop>
  <tabstop>nopeus_kentta_hex</tabstop>
  <tabstop>paino_kentta_si</tabstop>
  <tabstop>paino_valinta_si</tabstop>
  <tabstop>paino_kentta_sk</tabstop>
  <tabstop>paino_valinta_sk</tabstop>
  <tabstop>paino_kentta_hex</tabstop>
  <tabstop>lampo_kentta_si</tabstop>
  <tabstop>lampo_valinta_si</tabstop>
  <tabstop>lampo_kentta_sk</tabstop>
  <tabstop>lampo_valinta_sk</tabstop>
  <tabstop>lampo_kentta_hex</tabstop>
  <tabstop>sahko_kentta_si</tabstop>
  <tabstop>sahko_valinta_si</tabstop>
  <tabstop>sahko_kentta_sk</tabstop>
  <tabstop>sahko_valinta_sk</tabstop>
  <tabstop>sahko_kentta_hex</tabstop>
  <tabstop>aika_kentta_si</tabstop>
  <tabstop>aika_valinta_si</tabstop>
  <tabstop>aika_kentta_sk</tabstop>
  <tabstop>aika_valinta_sk</tabstop>
  <tabstop>aika_kentta_hex</tabstop>
  <tabstop>aika_nappi</tabstop>
  <tabstop>pvm__kentta_vvvv</tabstop>
  <tabstop>pvm__kentta_kk</tabstop>
  <tabstop>pvm__kentta_pp</tabstop>
  <tabstop>pvm__kentta_hh</tabstop>
  <tabstop>pvm__kentta_mm</tabstop>
  <tabstop>pvm__kentta_ss</tabstop>
  <tabstop>pvm__nappi_viimeinen</tabstop>
  <tabstop>pvm_sy_kentta_v</tabstop>
  <tabstop>pvm_sy_kentta_kk</tabstop>
  <tabstop>pvm_sy_kentta_pp</tabstop>
  <tabstop>pvm_sy_kentta_hh</tabstop>
  <tabstop>pvm_sy_kentta_mm</tabstop>
  <tabstop>pvm_sy_kentta_ss</tabstop>
  <tabstop>pvm_sy_nappi_viimeinen</tabstop>
  <tabstop>pvm_syt_kentta_vvvv</tabstop>
  <tabstop>pvm_syt_kentta_ppp</tabstop>
  <tabstop>pvm_sk_kentta_v</tabstop>
  <tabstop>pvm_sk_kentta_kk</tabstop>
  <tabstop>pvm_sk_kentta_pp</tabstop>
  <tabstop>pvm_sk_kentta_tm</tabstop>
  <tabstop>pvm_sk_kentta_rm</tabstop>
  <tabstop>pvm_sk_kentta_mm</tabstop>
  <tabstop>pvm_sk_kentta_nm</tabstop>
  <tabstop>pvm_sk_kentta_m</tabstop>
  <tabstop>pvm_sk_nappi_viimeinen</tabstop>
  <tabstop>pvm_skhex_kentta_v</tabstop>
  <tabstop>pvm_skhex_kentta_kk</tabstop>
  <tabstop>pvm_skhex_kentta_pp</tabstop>
  <tabstop>pvm_skhex_kentta_tm</tabstop>
  <tabstop>pvm_skhex_kentta_rm</tabstop>
  <tabstop>pvm_skhex_kentta_mm</tabstop>
  <tabstop>pvm_skhex_kentta_nm</tabstop>
  <tabstop>pvm_skhex_kentta_m</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
